neuro_san/registries/music_nerd_pro_multi_agents.hocon

```diff
@@ -0,0 +1,140 @@
+
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+# The schema specifications for this file are documented here:
+# https://github.com/cognizant-ai-lab/neuro-san/blob/main/docs/agent_hocon_reference.md
+
+{
+    "llm_config": {
+        "model_name": "gpt-4o",
+    },
+    "tools": [
+        # These tool definitions do not have to be in any particular order
+        # How they are linked and call each other is defined within their
+        # own specs.  This could be a graph, potentially even with cycles.
+
+        # This first agent definition is regarded as the "Front Man", which
+        # does all the talking to the outside world/client.
+        # It is identified as such because it is either:
+        #   A) The only one with no parameters in his function definition,
+        #      and therefore he needs to talk to the outside world to get things rolling.
+        #   B) The first agent listed, regardless of function parameters.
+        #
+        # Some disqualifications from being a front man:
+        #   1) Cannot use a CodedTool "class" definition
+        #   2) Cannot use a Tool "toolbox" definition
+        {
+            "name": "MusicNerdPro",
+
+            # Note that there are no parameters defined for this guy's "function" key.
+            # This is the primary way to identify this tool as a front-man,
+            # distinguishing it from the rest of the tools.
+
+            "function": {
+
+                # The description acts as an initial prompt.
+                "description": """
+I can help with music-related inquiries.
+"""
+            },
+
+            "instructions": """
+You're MusicNerdPro. Your job is to answer music-related questions and keep track of the "running cost".
+
+Each time you receive a question:
+1. Call the tool "MusicGuru" to get the answer to the music question. Pass the question to it as the "query" parameter.
+2. Call the tool "Accountant" to calculate the updated running cost. Pass it the current "running_cost" value. If this is the first question or no value has been stored yet, assume it is 0.0.
+3. Update your stored "running_cost" with the new value returned by the Accountant.
+
+Then, return a JSON message with two keys:
+- "answer": a string containing the response you received from MusicGuru.
+- "running_cost": a float representing the updated running cost from Accountant.
+""",
```

Comment:
Ah. Interesting break-up.
@vince-leaf : Is this what was needed for Ollama?
Or was this necessary for bedrock? (Seems like it shouldn't have been for bedrock).